migrations/sql-migrations/Makefile

```diff
@@ -0,0 +1,13 @@
+install:
+	@python3 -m venv venv
+	@venv/bin/pip3 install -r requirements.txt
+	@npm ci
+
+migrate-prod-to-dev:
```

Comment:
I would make this run `dynamo_to_json` only if the jsons aren't stored already inside `downloaded_data` becuase that's the whole point of splitting up the download and the upload to sql.

And perhaps add a special flag to force the redownload.

Is there a way we could have `migrate-dev-to-dev` and `migrate-prod-to-dev` in one command `migrate-to-dev` and use a parameter for the environemnt?

Reply:
Will have to see if make takes in variable.